tests/integration/test_trigger.py

```diff
@@ -38,3 +39,35 @@ async def test_trigger_invalid(mod_one, start, log_filter, flow_strs):
         log.clear()
         assert mod_one.pool.force_trigger_tasks(['*'], flow_strs) == 0
         assert len(log_filter(log, level=logging.WARN)) == 1
+
+
+async def test_trigger_no_flows(mod_one, start, log_filter):
+    """Test triggering a task with no flows present.
+
+    It should get the flow numbers of the most recent active tasks.
+    """
+    async with start(mod_one):
```

Comment:
The `mod_` bit scopes the fixture to the module, so the fixture won't get "reset" before being passed to the next test. The `mod_one` fixture is good for tests which poke a workflow but don't mutate it e.g. testing GraphQL responses.

```suggestion
async def test_trigger_no_flows(one, start, log_filter):
    """Test triggering a task with no flows present.

    It should get the flow numbers of the most recent active tasks.
    """
    async with start(one):
```